BP/common/culture/cultures/BP_burman.txt

```diff
@@ -0,0 +1,31 @@
+﻿#LT-Rascek: Taken from herkles "Dharmachakra"
+#and modified
+#
+#Usage Note:
+#Latest Version: 1.3.0 "Vishnu" (Released 26/12/2020)
+#
+#Dharmachakra is a flavor mod for CK3 focused on the Indian Subcontient and the neighboring regions of Burma and Tibet, as well as the Dharmic Religions of Buddhism, Jainism, and Hinduism and religions influenced by them.
+#
+#Feel free to use my work in your mods - all I ask is that I am credited and (if possible) to give me a heads up.
+
+rakhine = {# Rakhine People
+	color = { 0.6 0.3 0.8 }
+	
+	## FIXME!
```

Comment:
Is this going to get fixed in a later pull request?

Reply:
Eventually; the Burmese didn't migrate onto the map until much later (Nanzhao raiding in the 700s leading to the settlement of the Bamar in the 800s). The Rakhine people are a subset of the Burmese people, and should probably be a later cultural divergence (maybe 800s). So it'll be dealt with when we get to the point of including the Burmese migrations.